amplitude/waveDescription.cc

```diff
@@ -1029,7 +1029,7 @@ waveDescription::writeKeyFile(FILE&                      outStream,
 bool
 waveDescription::writeKeyFile(FILE&                  outStream,
                               const isobarAmplitude& amplitude,
-                              const bool             writeProdVert)
+                              const bool                       )
```

Comment:
parameter should not be removed but probably be added to the call to setKeysFromAmplitude in line 1036